src/modules/better-table/quill-better-table.ts

```diff
@@ -161,6 +165,21 @@ export default class BetterTable extends Module {
     });
   }
 
+  // doesn't affect line formatting - lists, bullets, which breaks tables in original function
+  removeTextFormatting(range = this.quill.getSelection()) {
+    if (range) {
+      this.quill.formatText(range, {
+        bold: false,
+        italic: false,
+        underline: false,
+        strike: false,
+        color: false,
+        background: false,
+        script: false,
+      });
+    }
+  }
```

Comment:
Tady bych řekl, že ti to třeba neresetuje velikost textu nebo zalinkování, ne? Podle mě mazání formátování tímhle způsobem nemůže fungovat nikdy 100%.